examples/color.yaml.template

```diff
@@ -359,4 +363,100 @@ spec:
             - name: "COLOR"
               value: "red"
 ---
-
+apiVersion: appmesh.k8s.aws/v1beta1
+kind: VirtualNode
+metadata:
+  name: tcpecho
```

Comment:
Long term, I think we should have multiple examples that each show off a subset of related features.  This will help users determine what configuration is necessary for each feature.  I think tcpecho is a good candidate for the first stand-alone example, what do you think?

Reply:
I believe examples should go to https://github.com/aws/aws-app-mesh-examples under walkthroughs.

Reply:
That makes sense.  I'll create an issue. We can leave it as one example for now, then.